src/main/java/org/gridsuite/sensitivityanalysis/server/dto/nonevacuatedenergy/NonEvacuatedEnergyStageDefinition.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.sensitivityanalysis.server.dto.nonevacuatedenergy;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.powsybl.iidm.network.EnergySource;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.sensitivityanalysis.server.dto.EquipmentsContainer;
+
+import java.util.List;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@Schema(description = "Sensitivity analysis non evacuated energy stage definition")
+public class NonEvacuatedEnergyStageDefinition {
+    List<EquipmentsContainer> generators;
+
+    EnergySource energySource;
+
+    @JsonProperty("pMaxPercents")
+    List<Float> pMaxPercents;
+}
```

Comment:
What if it becomes like that
```suggestion
public class NonEvacuatedEnergyGeneratorsSourceGroup {
    List<EquipmentsContainer> generators;

    EnergySource energySource;
}
```